spark-on-watson-studio/nested-json-to-table.md

```diff
@@ -0,0 +1,64 @@
+# Extract and save nested JSON elements into separate Db2 tables
+This example extracts nested elements from a JSON array and saves each array item into new Db2 tables.  
+
+**Note:** this is an introductory example and does not include any Db2 primary key-foreign key table relationships.
```

Comment:
Worth mentioning here that it doesn't recurse? (although it should be obvious to anyone reading the code)